docker/web/celery/worker/run.sh

```diff
@@ -16,7 +16,7 @@ echo "==> $(date +%H:%M:%S) ==> Setting up service... "
 python manage.py setup_service
 
 MAX_MEMORY_PER_CHILD="${WORKER_MAX_MEMORY_PER_CHILD:-2097152}"
-MAX_TASKS_PER_CHILD="${MAX_TASKS_PER_CHILD:-500000}"
+MAX_TASKS_PER_CHILD="${MAX_TASKS_PER_CHILD:-1000000}"
```

Comment:
What is the reason for this change?

Reply:
Prevent memory leaks when running a lot of tasks

Reply:
1M is a number that we should not reach soon, but my plan is to implement metrics with flower/grafana/prometheus to check that